doc/api/packages.md

```diff
@@ -924,6 +938,31 @@ require('./path/to/directory'); // This resolves to ./path/to/directory/main.js.
 When a package has an [`"exports"`][] field, this will take precedence over the
 `"main"` field when importing the package by name.
 
+### `"packageManager"`
+<!-- YAML
+added:
+  - REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+* Type: {string}
+
+```json
+{
+  "packageManager": "<package manager name>@<version>"
+}
+```
+
+The `"packageManager"` field defines which package manager is expected to be
+used when working on the current project. The npm client doesn't use this field
+at the moment, but defining it to any of the [supported package managers][]
+will let your teams use the exact same package manager versions without
+installing anything else than Node.js.
```

Comment:
I'm still a bit worried that this part could become outdated if npm were to release a new version with support for this field. @nodejs/npm do you know if this is something on the tracks?

Maybe we can rephrase it:
```suggestion
used when working on the current project. The npm client shipped with this version of
Node.js doesn't use this field, but defining it to any of the [supported package managers][]
will let your teams use the exact same package manager versions without
installing anything else than Node.js.
```

Reply:
Note that even if npm wants to use Corepack it'll require a small change in the tool (cf [this line of code](https://github.com/nodejs/corepack/blob/main/sources/types.ts#L18-L19)), which would require a PR on this repository - which would then update the documentation.

Reply:
I've updated the wording to simply point to the supported package manager page. I think it was the last "standalone" reference to which package managers were or weren't supported 🤔